internal/jobcontainers/jobcontainer.go

```diff
@@ -347,18 +306,56 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 	env = append(env, envs...)
 	env = append(env, sandboxMountPointEnvVar+"="+c.rootfsLocation)
 
-	// Add the rootfs location to PATH so you can run things from the root of the image.
+	path := os.Getenv("PATH")
```

Comment:
Are we getting the system's PATH in case we don't find a path in the environment used for starting the container?
Should we just set PATH to the values we append to `extraPaths` for consistency?
I had to read this a few times too, maybe add a comment to the code?

Reply:
I'm just gonna swap this for `var path string` and when I pass it to GetApplicationName just do `strings.Trim(path, "PATH=")`

Reply:
We should always have a PATH set due to grabbing a default env block above

Reply:
Changed

Reply:
This looks much cleaner now